nwb_conversion_tools/datainterfaces/axonadatainterface.py

```diff
@@ -0,0 +1,316 @@
+"""Authors: Steffen Buergers"""
+import os
+import re
+import datetime
+import contextlib
+import mmap
+import numpy as np
+
+import spikeextractors as se
+
+from pynwb import NWBFile
+from pynwb.behavior import Position, SpatialSeries
+
+from ..basedatainterface import BaseDataInterface
+from ..baserecordingextractorinterface import (
+    BaseRecordingExtractorInterface
+)
+
+from spikeextractors.extractors.nwbextractors.nwbextractors import check_module
```

Comment:
We actually just added a `get_module` util function to nwb-conversion-tools a while back. Can you refactor to include that instead?

Reply:
Ah, ok. Will do.